hack/e2e.sh

```diff
@@ -14,21 +14,20 @@ E2E_KUTTL_DIR=${E2E_KUTTL_DIR:-}
 # Defaults to empty string (run all tests)
 E2E_KUTTL_TEST=${E2E_KUTTL_TEST:-}
 
-kubectl kuttl test $E2E_KUTTL_DIR --test "$E2E_KUTTL_TEST"
+# Define a custom external network
+export E2E_EXTERNAL_NETWORK_NAME=${E2E_EXTERNAL_NETWORK_NAME:-public}
```

Comment:
If you look at the dalmatian run, the error is different:
```
      message: 'Expected HTTP response code [201 202] when accessing [POST http://10.1.0.45:9696/networking/v2.0/ports],
        but got 403 instead: {"NeutronError": {"type": "HTTPForbidden", "message":
        "Tenant f37f67b4bffd4136b3e32c471f53a574 not allowed to create port on this
        network", "detail": ""}}'
```

Devstack does not allow to create ports on the external subnet directly. I bet that if you change it from `public` to `private` as it was before, this will work.

Reply:
I think we need to put the on-failure trap back in the e2e tests. I wanted to check the reported state of these subnets while they were still (hopefully) in an error state, but they've been deleted already. All that's left in the logs are the objects created in config-local.